### DISCUSSION TITLE
Is there any way to get the float value that is the "Force on the sponge"?

### DISCUSSION THREAD

**Discussion by molegod** [empriselab/RCareWorld#57] [Other Q&A] [answered]
For the purposes of the hackathon, are we allowed to use the force on the sponge value? If so, what is the best way to do this?

Or should we make a robot skin to use as the force sensor? If it's the latter, is there a guide on how to mount the sensor onto the robot? As we don't have direct access to unity, so we're really unsure what is allowed in this hackathon or not :(

**Comment by YoruCathy** [comment, marked as answer]
Check the line with  comment "force on the sponge" in the starter script for bathing, feel free to use it.
